Doc/library/pyclbr.rst

```diff
@@ -29,6 +29,9 @@ modules.
    *path* is a sequence of directory paths prepended to ``sys.path``,
    which is used to locate the module source code.
 
+  This function is a filtered version kept for back compatibility which
```

Comment:
```suggestion
  This function is a filtered version kept for the backwards compatibility which
```

Reply:
'the' is wrong.  'back' is as good as 'backwards'.  I rewrote these two lines.